.github/workflows/priority-support-validation-prompt.yml

```
@@ -29,7 +29,7 @@ jobs:
         with:
           issue-number: ${{ github.event.issue.number }}
           body: |
-            You have created a priority support request ⏰. Please validate your support key using the link below:
+            You have created a support request under the paid ["Priority plan"](https://mui.com/legal/technical-support-sla/#priority-plan) ⏰. Please validate your support key using the link below:
```

I'm renaming these headers, it seems clearer like this:

```suggestion
            You have created a support request under the paid ["Priority Support plan"](https://mui.com/legal/technical-support-sla/#priority-support) ⏰. Please validate your support key using the link below:
```